IOPool/Streamer/src/StreamerFileIO.cc

```diff
@@ -4,6 +4,7 @@
 #include "FWCore/Utilities/interface/Adler32Calculator.h"
 #include "FWCore/Utilities/interface/Exception.h"
 
+using namespace edm::streamer;
 OutputFile::OutputFile(const std::string& name)
```

Comment:
I'd prefer
```suggestion
namespace edm::streamer {
  OutputFile::OutputFile(const std::string& name)
```
even if it leads to reformatting the file (which is fortunately short).

Reply:
done